src/common/schemas/specifications/payment-channel-claim.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "paymentChannelClaim",
+  "link": "payment-channel-claim",
+  "type": "object",
+  "properties": {
+    "channel": {
+      "$ref": "hash256",
+      "description": "256-bit hexadecimal channel identifier."
+    },
+    "amount": {
+      "$ref": "value",
+      "description": "XRP balance of this channel after claim is processed."
+    },
+    "balance": {
+      "$ref": "value",
+      "description": "Amount of XRP authorized by signature."
+    },
+    "signature": {
+      "$ref": "signature",
+      "description": "Signature of this claim."
+    },
+    "publicKey": {
+      "$ref": "publicKey",
+      "description": "Public key of the channel's sender"
+    },
+    "renew": {
+      "type": "boolean",
+      "description": "Clear the channel's expiration time."
+    },
+    "close": {
+      "type": "boolean",
+      "description": "Request to close the channel."
+    }
+  },
+  "required": ["channel"],
```

Comment:
Added some checks in https://github.com/ripple/ripple-lib/pull/745/files#diff-9efa2b0a32e1e12331fe1a8d45593513R36 this file, and tests. I was also wondering what happens when you don't specify the `cancelAfter` field on payment channel creation.